.travis.yml

```diff
@@ -17,20 +17,19 @@ addons:
 
 env:
   global:
-  - VERSION="12.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
-  - secure: axBDnKRsoWFK+yYseiC2VZ76aiKjRnT25p4R7wtMs0UXhydbJvac1lWEUWVm8XwiMKNK+sor3csVlm6mfuAqEAtBtC6mprS4oFgFdUSPoA0pPus99GLkOXJw6AD00R1p3O5S6ukQ1JNhMJZvIRhKFm+icd8+VJkn8GGclnuqrp4=
+  - VERSION="12.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0" MQT_DEP="PIP"
   matrix:
   - LINT_CHECK="1"
-  # Exclusion because of conflict with odoo 'account_facturx' module
-  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE="l10n_fr_account_invoice_facturx"
-  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="l10n_fr_account_invoice_facturx"
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+  - TESTS="1" ODOO_REPO="OCA/OCB"
 
 
 install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  # what we have here is not compatible with Odoo's account_facturx which is autoinstall
+  - rm -fr ${HOME}/{odoo,OCB}-12.0/addons/account_facturx
   - pip install unicodecsv unidecode ofxparse PyPDF2
```

Comment:
This `pip install` should not be necessary if python dependencies are properly declared in manifests.